src/types/field/attachmentField.ts

```diff
@@ -1,18 +1,14 @@
-import { IField, IFieldSchema } from './baseField'
+import {
+  AttachmentFieldBase,
+  AttachmentSize,
+  BasicField,
+} from '../../../shared/types/field'
 
-export enum AttachmentSize {
-  OneMb = '1',
-  TwoMb = '2',
-  ThreeMb = '3',
-  SevenMb = '7',
-  TenMb = '10',
-  TwentyMb = '20',
-}
+import { IFieldSchema } from './baseField'
 
-export interface IAttachmentField extends IField {
-  attachmentSize: AttachmentSize
-}
+export { AttachmentSize }
 
-export interface IAttachmentFieldSchema
-  extends IAttachmentField,
-    IFieldSchema {}
+export type IAttachmentField = AttachmentFieldBase
+export interface IAttachmentFieldSchema extends IAttachmentField, IFieldSchema {
+  fieldType: BasicField.Attachment
```

Comment:
this is unnecessary because the `Base` type already has the specific `fieldType` right?

Reply:
Needed, since IFieldSchema has `fieldType: BasicField`. This is reasserting that we want to use IAttachmentField's `fieldType`